### ISSUE TITLE
Windows 10 "Save as PDF" not working

### ISSUE THREAD

**Issue by Heikofant** [thawn/ttmp32gme#14]
I get the following error message when I hit the "Save as PDF"-Button on the printing subpage.

I am running Windows 10, Version 1607
```
C:\Users\HEIKOH~1\AppData\Local\Temp\par-4865696b6f20486f6c7a\cache-8be776d4409a3cf0c311f100f67ce391948b4a15\inc\lib\wkhtmltopdf.exe -B 0.5in -T 0.5in -L 0.5in -R 0.5in http://localhost:10020/pdf C:\Users\HiddenUser\AppData\Roaming\ttmp32gme\library\print.pdf
Qt: Untested Windows version 10.0 detected!
Error: This version of wkhtmltopdf is build against an unpatched version of QT, and does not support more then one input document.
Exit with code 1, due to unknown error.
```
`
unhandled callback exception on event (/print.pdf, AnyEvent::HTTPD=HASH(0x90dc648), AnyEvent::HTTPD::Request=HASH(0x98fdfb0)): Can't read C:\Users\HiddenUser\AppData\Roaming\ttmp32gme\library\print.pdf: No such file or directory at TTMp32Gme/LibraryHandler.pm line 176.`

**Comment by thawn** [comment]
The problem occurs because you have a space in your username. I'll try 
to fix this in the next version (together with a problem with unicode 
characters in the username). Until then try to create a user without 
spaces or special characters and use that account for ttmp32gme.

**Comment by hardmanj** [comment]
Hi,
i think it´s not a "space" Problem. Tested on different Win10-Systems. One with Space in the UserName. Two without.
Even the Username P599 didn´t work.

**Comment by thawn** [comment]
@hardmanj  do you get the same error message as Heikofant? If not could you please paste the output from the command window here?

**Comment by thawn** [comment]
meanwhile, printing directly from Edge should work also...

**Comment by thawn** [comment]
fixed in [v0.2.3](https://github.com/thawn/ttmp32gme/releases/tag/v0.2.3)
